apps/tailwind-components/components/content/type/ContentTypeString.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <p>{{ field.value }}</p>
+</template>
+
+<script setup lang="ts">
+import type { ISectionField } from "~/interfaces/types";
```

Comment:
Is this file available? To organise the files, should we use interfaces or types?